test/src/unit-capi-vfs.cc

```diff
@@ -54,36 +54,17 @@ struct VFSFx {
   // The unique directory object
   tiledb::sm::TemporaryLocalDirectory temp_dir_{"tiledb_test_"};
 
-  const std::string HDFS_TEMP_DIR = "hdfs://localhost:9000/tiledb_test/";
-  const std::string S3_PREFIX = "s3://";
-  const std::string S3_BUCKET = S3_PREFIX + random_name("tiledb") + "/";
-  const std::string S3_TEMP_DIR = S3_BUCKET + "tiledb_test/";
-  const std::string AZURE_PREFIX = "azure://";
-  const std::string AZURE_CONTAINER =
-      AZURE_PREFIX + random_name("tiledb") + "/";
-  const std::string AZURE_TEMP_DIR = AZURE_CONTAINER + "tiledb_test/";
-  const std::string FILE_TEMP_DIR =
-#ifndef _WIN32
-      "file://" +
-#endif
-      temp_dir_.path();
-  const std::string MEMFS_TEMP_DIR = std::string("mem://tiledb_test/");
-
   // TileDB context and vfs
   tiledb_ctx_t* ctx_;
   tiledb_vfs_t* vfs_;
 
   // Functions
   VFSFx();
   ~VFSFx();
-  void check_vfs(const std::string& path);
-  void check_write(const std::string& path);
-  void check_move(const std::string& path);
-  void check_copy(const std::string& path);
-  void check_read(const std::string& path);
-  void check_append(const std::string& path);
-  void check_ls(const std::string& path);
-  static std::string random_name(const std::string& prefix);
+
+  inline void require_tiledb_ok(capi_return_t rc) const {
+    tiledb::test::require_tiledb_ok(ctx_, (int)rc);
```

Comment:
Once you use ordinary_context, just use tiledb::test::require_tiledb_ok directly everywhere.